### ISSUE TITLE
Pretrained model for SingleViewModel

### ISSUE THREAD

**Issue by mr-Mojo** [mworchel/svbrdf-estimation#45]
Hi, 
thanks for this cool PyTorch-port! Do you have a trained checkpoint for the SingleView Model available? That'd be great! 

Thanks, and best

**Comment by mworchel** [comment]
Hi,

unfortunately, I don't have a trained checkpoint I can share. However, it should be pretty straightforward to train the model with the single-view dataset.

Personally, if you want to build a research project on this method, I would recommend using the reference implementation, which also comes with a pre-trained model: https://github.com/valentin-deschaintre/Single-Image-SVBRDF-Capture-rendering-loss
